pkg/plugins/runtime/k8s/webhooks/injector/injector_test.go

```diff
@@ -656,6 +624,96 @@ spec:
 		}),
 	)
 
+	DescribeTable("should not inject Kuma into a Pod",
+		func(given testCase) {
+			// setup
+			inputFile := filepath.Join("testdata", fmt.Sprintf("inject.%s.input.yaml", given.num))
+			goldenFile := filepath.Join("testdata", fmt.Sprintf("inject.%s.golden.yaml", given.num))
```

Comment:
Could we change the numbering to start at 01 and rename files from inject to skip_inject for example? Would make it easier to find failing test files

Reply:
sure, seems sensible